compiler/noirc_frontend/src/tests.rs

```diff
@@ -1733,7 +1733,7 @@ fn numeric_generic_as_param_type() {
     ));
     // Error from the return type
     assert!(matches!(
-        errors[2].0,
+        errors[1].0,
```

Comment:
We've got 2 assertions on the same error now.

Reply:
Fixed: it's still passing locally, thankfully